control-plane/agents/src/bin/core/main.rs

```diff
@@ -110,6 +110,14 @@ pub(crate) struct CliArgs {
     /// This is useful when the frontend nodes do not support the NVMe ANA feature.
     #[clap(long, env = "HA_DISABLED")]
     pub(crate) disable_ha: bool,
+
+    /// Formatting style to be used while logging.
+    #[clap(global = true, default_value = FmtStyle::Pretty.as_ref(), short, long)]
```

Comment:
Why do we need global here?

```suggestion
    #[clap(default_value = FmtStyle::Pretty.as_ref(), short, long)]
```